src/pages/Background/index.js

```diff
@@ -1,2 +1,66 @@
+//Even while development.. extension need to be reloaded for changes here to come into effect
+import { closest, distance } from 'fastest-levenshtein';
+
 console.log('This is the background page.');
+
 console.log('Put the background scripts here.');
+
+console.log('new line by hamza');
+
+//fetch('http://localhost:5000').then((res) => res.json()).then((res) => { console.log(JSON.stringify(res)) })
+
+// chrome.runtime.onInstalled(() => {
+//     chrome.runtime.onMessage.addListener((msg, sender, res) => {
+//         alert(msg)
+//     })
+// })
+
+let symbolData;
+
+fetch('https://script.google.com/macros/s/AKfycbxJPjCcowBQS124lLZfmw1ItprLtxNx7MiAmTgsdcp_dpTazJJVM7zEOrM_KHPSoib4/exec', {
+    method: 'GET',
+    mode: 'cors'
+}).then(res => res.json()).then((data) => {
+    console.log(data)
+    symbolData = data
+})
```

Comment:
why not store it in user's local storage, rather refetching in background every time, if there is update we can update it

Reply:
sure we can, but it's not complete yet.. so a lot of things are not optimized rn.

Reply:
yes, i had the idea, but i have not optimized everything yet.
just wanted the base logic to work

Reply:
rather than using raw `localstorage` use something like `dexie-db`. Your life will be much easier.